speech/speech_synthesis_result.go

```diff
@@ -21,8 +21,8 @@ import "C"
 type SpeechSynthesisResult struct {
 	handle C.SPXHANDLE
 
-	// ResultID specifies the result identifier.
-	ResultID string
+	// Result ID specifies the result identifier.
+	ResultId string
```

Comment:
We are using ID everywhere else (I think the linter complained otherwise). Also, for refdoc comments the name of the member should match the name in the doc comment.

Reply:
thanks,  change back to `ResultID `